x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_list.ts

```diff
@@ -0,0 +1,146 @@
+    allow_lazy_open: false,
+    model_snapshot_retention_days: 0,
+    results_index_name: '',
+    job_id: jobsSummaryId,
+    description: 'mean(responsetime) partition=airline on farequote dataset with 1h bucket span',
+    groups: ['farequote', 'automated', 'multi-metric'],
+    analysis_config: {
+      bucket_span: '1h',
+      influencers: ['airline'],
+      detectors: [
+        { function: 'mean', field_name: 'responsetime', partition_field_name: 'airline' },
+      ],
+    },
+    data_description: { time_field: '@timestamp' },
+    analysis_limits: { model_memory_limit: '50mb' },
+    model_plot_config: { enabled: true },
+  };
+
+  describe('job list', function () {
+    this.tags(['ml']);
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.createDataViewIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      const JOB_CONFIG = ml.commonConfig.getADFqSingleMetricJobConfig(jobId);
+      const DATAFEED_CONFIG = ml.commonConfig.getADFqDatafeedConfig(jobId);
+
+      await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+
+      await ml.api.waitForJobState(jobId, JOB_STATE.CLOSED);
+      await ml.api.openAnomalyDetectionJob(jobId);
+
+      const { body, status } = await supertest
+        .post(`/internal/ml/anomaly_detectors/${jobId}/_forecast`)
+        .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+        .set(getCommonRequestHeader('1'))
+        .send({ duration: '1d' });
+      ml.api.assertResponseStatusCode(200, status, body);
+
+      await ml.api.assertForecastResultsExist(jobId);
+
+      await ml.api.createAnomalyDetectionJob(multiMetricJobConfig);
+
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToSettings();
+      await ml.settings.navigateToCalendarManagement();
+
+      await ml.settingsCalendar.assertCreateCalendarButtonEnabled(true);
+      await ml.settingsCalendar.navigateToCalendarCreationPage();
+
+      await ml.settingsCalendar.waitForFormEnabled();
+
+      await ml.settingsCalendar.assertJobSelectionExists();
+      await ml.settingsCalendar.assertJobSelectionEnabled(true);
+      await ml.settingsCalendar.assertJobGroupSelectionExists();
+      await ml.settingsCalendar.assertJobGroupSelectionEnabled(true);
+
+      await ml.settingsCalendar.setCalendarId(calendarId);
+
+      await ml.settingsCalendar.selectJob(jobId);
+
+      await ml.settingsCalendar.saveCalendar();
+      await ml.settingsCalendar.assertCalendarRowExists(calendarId);
+
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await ml.navigation.navigateToMl();
+
+      await ml.navigation.navigateToJobManagement();
+    });
+
+    after(async () => {
+      await ml.api.closeAnomalyDetectionJob(jobId);
+      await ml.api.cleanMlIndices();
+      await ml.api.deleteAnomalyDetectionJobES(jobId);
```

Comment:
These two row should change order: we should first delete the job before we clean everything.